README.md

```diff
@@ -458,7 +458,7 @@ zoomer: {
     markerHeight: 20,
 
     // canvas
-    residueFont: "13", //in px
+    residueFont: "13px Helvetica Neue",
```

Comment:
Also changed the default to `Helvetica Neue`.

Sequence Viewer will however likely use a custom font like "Objective".